bin/pacbio_barcode_check.py

```diff
@@ -45,19 +47,28 @@ def detect_barcodes_from_read_file_names(barcodes_fasta_path, pacbio_read_files)
 
 def check_if_barcodes_exist_in_barcodes_fasta(barcodes_list, barcodes_fasta_path):
     """
-    Checks if the specified barcodes exist in the barcode sequences FASTA file, exits with an error message if a barcode is not found
+    Checks if the specified barcodes exist in the barcode sequences FASTA file, prints a message for each missing barcode.
```

Comment:
I think to fix the GitHub testing error, `pacbio_multiplexing_barcode_names: "bc1008,bc1009"` in [https://github.com/sanger-tol/ascc/blob/vecscreen2/assets/github_testing/test.yaml](https://github.com/sanger-tol/ascc/blob/vecscreen2/assets/github_testing/test.yaml) should be changed to `pacbio_multiplexing_barcode_names: "bc2008,bc2009"` because these ones are present in the [https://www.pacb.com/wp-content/uploads/SMRTbell_Barcoded_](https://www.pacb.com/wp-content/uploads/SMRTbell_Barcoded_%20Adapter_Plate_3.0_bc2001-bc2096.fasta_.zip)
[Adapter_Plate_3.0_bc2001-bc2096.fasta_.zip](https://www.pacb.com/wp-content/uploads/SMRTbell_Barcoded_%20Adapter_Plate_3.0_bc2001-bc2096.fasta_.zip) file that is used for the test. `"bc1008,bc1009"` aren't present, which triggers the error

Reply:
Reverted to previous version of script

Reply:
Updated test.yamls with new values